migration/src/lib/akumina-site.js

```diff
@@ -101,16 +108,22 @@ class AkuminaSite {
     const list = sp.web.lists.getByTitle(listName);
     let itemsImported = 0;
     let batchExecuted = false;
+    let addedContent = 0;
     while ((itemsImported < items.length) || !batchExecuted) {
       batchExecuted = false;
       const batch = sp.createBatch();
       for (let i = 0; (i < batchSize) && (itemsImported < items.length); i++) {
+        if(items[itemsImported].metadata.hasOwnProperty('MigrateAdminUse') &&
```

Comment:
Is this just a safety check? I assume we're not exporting content that doesn't have this flag enabled.